orchestrator/ethereum_oracle/src/oracle_resync.rs

```diff
@@ -108,10 +108,7 @@ pub async fn get_last_checked_block(
         current_block = end_search;
     }
 
```

Comment:
lets add a todo or a task to do the first deposit as soon as we deploy and count that as the first event

Reply:
done